Xamarin.Forms.Xaml/MarkupExtensions/DataTemplateExtension.cs

```diff
@@ -2,17 +2,26 @@
 
 namespace Xamarin.Forms.Xaml
 {
-	[ContentProperty("Type")]
+	[ContentProperty("TypeName")]
+	[ProvideCompiled("Xamarin.Forms.Build.Tasks.DataTemplateExtension")]
 	public sealed class DataTemplateExtension : IMarkupExtension<DataTemplate>
 	{
-		public string Type { get; set; }
+		public string TypeName { get; set; }
```

Comment:
that's the API-Change

Reply:
Why did we change this? It looks like we are talking about the type... Although the other extensions are using TypeName it appears... So I am assuming that?

Reply:
1. to be aligned with the `{x:Type}` markup
2. to avoid confusion, and require a type name, not an actual type, so this is not valid: `{DataTemplate {Type local:foo}}`